### DISCUSSION TITLE
Share about how you found this repository

### DISCUSSION THREAD

**Discussion by CoderRC** [CoderRC/libmingw32_extended#4] [General]
Share about how you found this repository to help reach this project to more people
@stars

**Comment by kassane** [comment]
Randomly searching for posix libraries to mingw until found your project and forked to adding it as library to zig-clang/mingw.
My fork - CI test
https://github.com/kassane/libmingw32_extended/actions/runs/4640131129/jobs/8211735057

**Comment by dailker** [comment]
We met CoderRC on Discord and he was really passionate about his project and after that I started creating PR regularly.
